forge-gui/res/editions/Wilds of Eldraine Commander.txt

```diff
@@ -181,22 +182,19 @@ ScryfallCode=WOC
 173 U Vitu-Ghazi, the City-Tree @Martina Pilcerova
 
 [tokens]
-0 role_monster @Unknown
-0 role_royal @Unknown
-0 role_sorcerer @Unknown
-0 role_virtuous @Unknown
-0 role_virtuous @Unknown
-0 role_virtuous @Unknown
-5 w_2_2_cat @April Prime
-6 w_1_1_cat_lifelink @Filip Burburan
+1 role_monster @Rovina Cai
+# Need to flip it
+1 role_virtuous @Rovina Cai
+2 role_royal @Rovina Cai
+# Need to flip it
+2 role_virtuous @Rovina Cai
 7 w_1_1_human_soldier @Kimonas Theodossiou
-8 w_2_4_ox @Aaron Miller
 9 w_2_2_pegasus_flying @Lars Grant-West
 10 u_1_1_faerie_flying @Irina Nordsol
 11 b_1_1_faerie_rogue_flying @Dave Allsop
 12 r_4_2_pirate_noblock @Igor Krstic
-13 g_3_3_elephant @Lars Grant-West
-14 g_1_1_human_monk_g @Anastasia Ovchinnikova
 15 g_1_1_saproling @Joseph Meehan
 16 ub_1_1_faerie_rogue_flying @E. M. Gist
-17 wb_1_1_spirit_flying @Josh Hass
```

Comment:
i really hate it when these numbers are missing
but it looks like that was an issue with wizards too:
https://magic.wizards.com/en/news/card-preview/the-tokens-of-wilds-of-eldraine